Health/src/main/java/com/sciencesquad/health/health/Nutrition/RealmNutritionModule.java

```diff
@@ -0,0 +1,172 @@
+         * :)
+         */
+
+        setRealmName("nutrition.realm");
+        setRealmContext(BaseApplication.application().getApplicationContext());
+        init();
+    }
+
+    @Override
+    public void init(){
+
+        /**
+         * This sets up the Realm for the module.
+         *
+         * Notes:
+         *      - sets up a RealmConfiguration to establish the realm.
+         *      - uses RealmConfiguration to build the Realm for this module.
+         *      - Sets up a RealmList to hold all the RealmModels in a list for history purposes.
+         *      - configNutritionRealm is set to delete the Realm if there are changes in the model,
+         *      which in the early stages of development is okay.
+         *      HOWEVER, once this is released, we need to make a separate Migration, which is will
+         *      support multiple versions of the application.
+         */
+
+        configNutritionRealm = new RealmConfiguration.Builder(getRealmContext())
+                .name(getRealmName())
+                .setModules(this)
+                .deleteRealmIfMigrationNeeded()
+                .build();
+
+        realm = Realm.getInstance(configNutritionRealm);
+        nutritionModels = new RealmList<>();
+    }
+
+    @Override
+    public void update(){
+
+        /**
+         * This will update the realm asynchronously from the UI thread
+         * and will update the model given that something has changed.
+         * This will generate an event to all subscribers on the Event Bus.
+         *
+         * This can later be modified to write to a list of Realm Objects
+         * which could be useful for a history.
+         */
+        realm.beginTransaction();
+        realm.copyToRealmOrUpdate(nutritionModels);
+        realm.commitTransaction();
+
+        BaseApplication.application().eventBus().publish(RealmUpdateEvent.from(this).key(returnRealmKey()).create());
+    }
+
+    @Override
+    public void query(){
+        /**
+         *
+         * This will set up a Realm Query object
+         * based off of the Realm Object associated with the module
+         *
+         * From there, one can do SQL-esq queries which returns a
+         * RealmList<RealmObjectClass> results,
+         * which is pertinent to that query.
+         */
+
+        queryNutrition = realm.where(RealmNutritionModel.class);
+
+    }
+
+    @Override
+    public void clearRealm(){
+        /**
+         *
+         * This will clear all the relevant models from the realm.
+         * This will generate an event to all subscribers on the Event Bus.
+         * Use this with caution.
+         */
+
+        realm.beginTransaction();
+        realm.clear(RealmNutritionModel.class);
+        realm.commitTransaction();
+
+        BaseApplication.application().eventBus().publish(RealmEmptyEvent.from(this).RealmName(getRealmName()).create());
+    }
+
+    @Override
+    public void closeRealm() {
+
+        /**
```

Comment:
Comments go above the method, not inside; you might be thinking of Python :wink: